index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+      </div>
+    </div>
+    <div class="card_cnt work-3">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-4">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-5">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-6">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  </section>
+
+  <!-- Section About -->
+  <section class="about back_dark">        
+    <div class="about_container">
+      <div class="about_text">
+        <h3 class="Title1 Tittle_about">About me</h3>
+        <h4 class="Simpletext">Hello I'm a software developer! I can help you build a product, feauture or website. Look through some of my work and expererience!. If you like what you see and have a prject you need coded, don't hesitate to contact me.</h4>
+        <a class="project_btn_all project_btn_1 resume_btn" href="">Get My Resume</a>
+      </div>            
+    </div>  
+  </section>
+  
+  <!-- Section Skills -->
+  <section class="back_dark">
```

Comment:
In between these 2 sections consider using the same styles like there is a horizontal line that divides the section and its part of the about section. Moreover, there is no white background in the styles so consider rectifying it. I saw you added a  long margin so consider using padding instead.

## Your Design:

![image](https://user-images.githubusercontent.com/94223367/227764079-64d97736-100c-477e-8f10-d9d2bfa3313d.png)


## Figma Design:

![image](https://user-images.githubusercontent.com/94223367/227764082-be3ee841-7f13-4687-bc04-13b942ed95b0.png)

Reply:
Done. thank you @ABDULALI3468